src/SFA.DAS.EmploymentCheck.Functions/Application/Services/Hmrc/HmrcService.cs

```diff
@@ -144,5 +156,16 @@ private async Task RetrieveAuthenticationToken()
         }
 
         #endregion RetrieveAuthenticationToken
+
+        #region AbandonRelatedRequests
```

Comment:
what's the region for?? it's polluting our code-base.

see https://marcduerst.com/2016/10/03/c-regions-are-evil/

Reply:
Removed regions.